applets/notification_area/status-notifier/sn-item-v0.c

```diff
@@ -895,6 +908,8 @@ g_signal_cb (GDBusProxy *proxy,
     new_title_cb (v0);
   else if (g_strcmp0 (signal_name, "NewIcon") == 0)
     new_icon_cb (v0);
+ else if (g_strcmp0 (signal_name, "XAyatanaNewLabel") == 0)
+   new_label_cb (v0, parameters);
```

Comment:
please, fix the indentation.

Reply:
@rbuj  To tell you the truth, I'm really bothered by the indent format of mate。It takes more time to modify indentation than to modify code，Is the indent here two spaces？

Reply:
Where did you see a single blank space being used for indentation?

Reply:
> To tell you the truth, I'm really bothered by the indent format of mate。

And i am really bothered to mark always the same formatting problems by people.